pkg/bgpv1/manager/instance/fake_instance.go

```diff
@@ -10,8 +10,8 @@ import (
 // NewFakeBGPInstance is fake BGP instance, to be used in unit tests.
 func NewFakeBGPInstance() *BGPInstance {
 	return &BGPInstance{
-		Config:   nil,
-		Router:   types.NewFakeRouter(),
-		Metadata: make(map[string]any),
+		Config: nil,
+		Router: types.NewFakeRouter(),
+		Global: types.BGPGlobal{},
```

Comment:
I suggest having separate commit for introducing Global config, would be easier to review what it impacts.

Reply:
split the change into 2 commits